src/collective/cover/tiles/calendar.py

```diff
@@ -107,15 +124,6 @@ def getYearAndMonthToDisplay(self):
         year = request.get('year', None)
         month = request.get('month', None)
 
-        # Next get the data from the SESSION
-        if self.calendar.getUseSession():
```

Comment:
why not? this calendar tile don't need to use session to keep the year and month as parameter. I'll just use GET parameters to do it.